src/models/player.js

```diff
@@ -11,6 +11,7 @@ export default class Player {
     this.showMoves = false
     this.selectedPiece = null
     this.game = game
+    this.promotedPawns = []
```

Comment:
also probably not necessary. I think it would be simpler to work with this if they go into the `capturedPieces`, but this feedback is a bit nitpicky